lang/java/reef-tang/tang-test-jarA/src/main/java/org/apache/reef/tang/examples/A.java

```diff
@@ -19,5 +19,4 @@
 package org.apache.reef.tang.examples;
 
 public class A {
-  private int iAmAClass = 0;
```

Comment:
Hi, I think we need to keep this since this is an example to show `Tang`'s behavior. This is not a bug, isn't it?